dist/index.js

```diff
@@ -6193,7 +6193,7 @@ async function getOTPVersion(otpSpec0, osVersion) {
   const otpVersions = await getOTPVersions(osVersion)
   const otpSpec = otpSpec0.match(/^(OTP-)?([^ ]+)/)
   let otpVersion
-  if (otpSpec[1]) {
+  if (otpSpec[1] && !isStrictVersion()) {
```

Comment:
This was just us being super strict with input, but for the edge case being tested we do want to allow "Using -otp- or OTP-", even though it is to be made explicit via the version-type option.